### ISSUE TITLE
Problem with inappbrowser with cca

### ISSUE THREAD

**Issue by GustavoCostaW** [MobileChromeApps/mobile-chrome-apps#552]
I'm using the inappbrowser api, but I can not open any site within my app with cca, I do not want to open in the browser, but within my app with a dialog like the old childbrowser, someone could help me?

**Comment by GustavoCostaW** [comment]
@agrieve sorry for delay, but not work for me :(

I have the plugin in project
![screen shot 2015-04-25 at 11 17 13](https://cloud.githubusercontent.com/assets/874909/7333225/5622f7b2-eb3d-11e4-8249-76df5b1e4999.png)

But in console for open URL, the method InAppBrowser in cordova object not exists
![screen shot 2015-04-25 at 11 21 58](https://cloud.githubusercontent.com/assets/874909/7333226/562471f0-eb3d-11e4-890c-3e804ef41ff5.png)


Can you help me?

**Comment by agrieve** [comment]
Yeah, not sure what's going on. Could you try updating to the latest cca (0.7.0) via `npm install -g cca` and see if it still happens?
